### DISCUSSION TITLE
Mismatch rank and MMR

### DISCUSSION THREAD

**Discussion by Frankpro42** [Heroes-Profile/heroesprofile#936] [Ideas]
As an example [this player](https://www.heroesprofile.com/Player/Harukage/3462590/2) is master on both QM and SL, but his MMR is 2,450 and 2,604 respectively while the norm should currently be 2,812 and 2,948 to be master.

**Comment by Zemill** [comment]
Closing this as it fits more into the bug reporting section

see https://github.com/Heroes-Profile/heroesprofile/issues/937
